whisper_online.py

```diff
@@ -152,7 +152,8 @@ def set_translate_task(self):
 class OpenaiApiASR(ASRBase):
     """Uses OpenAI's Whisper API for audio transcription."""
 
-    def __init__(self, lan=None, temperature=0, logfile=sys.stderr):
+    def __init__(self, modelsize=0, lan=None, cache_dir=None, model_dir=None, temperature=0, logfile=sys.stderr):
+        # modelsize, lan, cache_dir, model_dir are not used, but they're here for compatibility with the other ASR classes
```

Comment:
thanks, but I don't agree. There shouldn't be the unused parameters modelsize, cache_dir, and model_dir. Users should use this class other way than the other

Reply:
lan is used

Reply:
>  I can change how this class is used in whisper_online_server instead.

yes, please

Reply:
the code for this is here: https://github.com/ufal/whisper_streaming/blob/c92d469597829ff5da9400b6dabe6014a364e8d7/whisper_online.py#L580C1-L594C69

Let's not duplicate it. Can you make it a function in whisper_online.py that would be called from server?